doc/api.yml

```diff
@@ -288,6 +291,10 @@ paths:
           description: "Filter students by last name, case is ignored"
           schema:
             type: string
+        - name: course_id
```

Comment:
Add a description.

Also add a TODO where we could filter by an array of course_id.